src/content/add-to-app/ios/project-setup.md

```diff
@@ -152,108 +156,103 @@ some/path/
     └── Podfile
 ```
 
-If your existing application (`MyApp`) doesn't
-already have a Podfile, run `pod init` in the  
-`MyApp` directory to create one. 
-You can find more details on using 
-CocoaPods in the [CocoaPods getting started guide][].
+#### Update your Podfile
 
-<ol>
-<li>
+1. If your existing application (`MyApp`) doesn't already have a Podfile,
+   use the `pod init` command in the `MyApp` directory to create one.
 
-Add the following lines to your `Podfile`:
+   To learn more about CocoaPods,
+   consult the [CocoaPods getting started guide][].
 
-```ruby title="MyApp/Podfile"
-flutter_application_path = '../my_flutter'
-load File.join(flutter_application_path, '.ios', 'Flutter', 'podhelper.rb')
-```
+1. Add the following lines to your `Podfile`.
```

Comment:
Can we clarify where line should be added?

This was unclear to me when I followed this guide. I ended up looking at our samples on GitHub to figure this out. See: https://docs.google.com/document/d/1fnmxAHTLhYTmssi8YI55av7PRXjFPNutOrEAxGrYIEI/edit?resourcekey=0-A9xndfJnQCTdRL0QtOqRig&tab=t.0#heading=h.uxxukmyfol1w

```suggestion
1. Add the following lines at the top of your `Podfile`.
```